README.md

```diff
@@ -16,3 +16,5 @@ the function and run automatically when you commit through a GitHub action.
 ## Runtime Analysis
 
 What is the worst-case big $\Theta$ time complexity of your algorithm?
+
+Answer: The worst case $\Theta$ time complexity for my algorithm would be $\Theta(V! * V_1 * V_2)$. This is because the recursive call for the helper function lies in two nested for loops that iterate through the graph1 vertices and the graph2 vertices. So the worst case would come from all possible permutations of mappings being explored. All time complexities of other functions are ignored because the dominant term is $(V! * V^2)$.
```

Comment:
So this is what happens in a single execution of the function. But the function is calling itself recursively. Doesn't that mean that the $V_1 * V_2$ term will appear over and over again?

Reply:
Yes it does. So would that be (V^2)!?

Reply:
You could write down the recurrence relation and solve it.